code/modules/mob/living/silicon/robot/login.dm

```diff
@@ -1,13 +1,15 @@
 /mob/living/silicon/robot/Login()
 	..()
 	regenerate_icons()
-	show_laws(0)
+	if(!isshell(src))
```

Comment:
Maybe do this in the shell's type? I don't even care if you have to copypaste, it's better than having src checks.